docker/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# See the file "COPYING" in the main distribution directory for copyright.
+
+# Layer to build Zeek. {{{
+FROM debian:buster-slim AS build_zeek
+
+# Configure system for build.
+RUN apt-get -q update \
+ && apt-get install -q -y --no-install-recommends \
+     bind9 \
+     bison \
+     cmake \
+     flex \
+     g++ \
+     gcc \
+     libmaxminddb-dev \
+     libpcap-dev \
+     libssl-dev \
+     libz-dev \
+     make \
+     python3-minimal \
+     python3-dev \
+     swig \
+     ninja-build \
+     python3-pip \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+COPY . /src/zeek
+WORKDIR /src/zeek
+RUN ./configure \
+     --generator=Ninja \
+     --build-type=Release \
+ && ninja -C build install
+
+# Install dependencies for zkg.
+RUN pip3 install --no-cache-dir GitPython semantic-version
+# }}}
+
+# Final layer containing all artifacts. {{{
+FROM debian:buster-slim AS final
+
+RUN apt-get -q update \
+ && apt-get install -q -y --no-install-recommends \
+     ca-certificates \
+     git \
+     libmaxminddb0 \
+     libpython3.7 \
+     libpcap0.8 \
+     libssl1.1 \
+     libz1 \
+     python3-minimal \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# Copy over Zeek installation.
+COPY --from=build_zeek /usr/local/zeek /usr/local/zeek
+ENV PATH "/usr/local/zeek/bin:${PATH}"
+
+# Copy over Python dependencies.
+COPY --from=build_zeek /usr/local/lib/python3.7/dist-packages /usr/local/lib/python3.7/dist-packages
+
+# Setup zkg.
```

Comment:
You can drop this part Benjamin. For the Zeek-bundled `zkg` installation, you no longer need to call this because that `zkg` comes preconfigured. The use case where you might still want this is for people who want `zkg --user`, i.e., forcing `zkg` to keep state in the home directory. But that's usually for folks who care about separating user roles/permissions, and I don't think that's in scope for the Docker image.

Reply:
I assume you meant the line `zkg autoconfig --force`? I'll drop it.

Reply:
Thanks for checking — I meant both of the lines. The bundled `zkg` automatically sources the configuration installed into $ZEEKROOT/etc/zkg/config`. The `autoconfig` step therefore isn't necessary. I think the `zkg env` doesn't add anything because it just provides the installation defaults. Do you think you need it? I might be missing something here but I never use it (with the bundled `zkg`).

Reply:
I was mainly after the modified `PATH`, but I see that this just contains the `PATH` we already add for `zeek`. I'll get rid of this (which is nasty anyway).